src/redux/reducers/userReducer.ts

```diff
@@ -26,6 +27,7 @@ export const userReducer = (state: User = emptyUser, action: UserActions | Orcid
         roles,
         application: action.user['custom:application'] as ApplicationName,
         isLoggedIn: true,
+        organizationId: getOrganizationIdByOrganizationNumber(action.user['custom:orgNumber']),
       };
```

Comment:
Syns egentlig mye her heller skulle vært gjort i action, men det er evt en separat sak

Reply:
agree to disagree man